tidy3d/components/spice/sources/ac.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+from typing import Optional
+
+import pydantic.v1 as pd
+
+from tidy3d.components.base import Tidy3dBaseModel
+from tidy3d.components.types import ArrayFloat1D
+from tidy3d.constants import HERTZ, VOLT
+from tidy3d.constants import inf as td_inf
+
+
+class SSACVoltageSource(Tidy3dBaseModel):
+    """
+    Small-signal AC voltage source.
+
+    This source represents a small-signal AC excitation defined by a list of bias voltages,
+    a set of small-signal analysis frequencies, and a small-signal amplitude (linear scale).
+
+    Notes
+    -----
+    - The bias `voltage` refer to the DC operating point above the simulation ground.
+    - The `frequency` array lists the AC analysis frequencies (in Hz).
+    - The `amplitude` is the small-signal AC magnitude (in Volts).
+    - Currently, electrical ports are not defined.
+
+    Examples
+    --------
+    >>> import tidy3d as td
+    >>> voltages = [-0.5, 0, 1, 2, 3, 4]
+    >>> frequency = [1e3, 1e4, 1e5]
+    >>> amplitude = 1e-3
+    >>> voltage_source = td.SSACVoltageSource(
+    ...     voltage=voltages,
+    ...     frequency=frequency,
+    ...     amplitude=amplitude,
+    ... )
+    """
+
+    name: Optional[str]
+
+    voltage: ArrayFloat1D = pd.Field(
+        ...,
+        title="DC Bias Voltages",
+        description="List of DC operating point voltages (above ground) used with 'VoltageBC'.",
+        units=VOLT,
+    )
+
+    frequency: ArrayFloat1D = pd.Field(
+        ...,
+        title="AC Analysis Frequencies",
+        description="List of small-signal analysis frequencies (in Hz).",
```

Comment:
No need to say the units in the description if they are defined in the `units` argument below, as they should be. See for example how this is parsed for the existing VoltageBC:

<img width="1611" height="108" alt="image" src="https://github.com/user-attachments/assets/05ef1c7f-f8ac-448e-8c75-cb5745c1dce7" />